homeassistant/components/elgato/light.py

```diff
@@ -36,6 +37,16 @@ async def async_setup_entry(
     info = await elgato.info()
     async_add_entities([ElgatoLight(elgato, info)], True)
 
+    platform = current_platform.get()
+    if not platform:
+        return
+
+    platform.async_register_entity_service(  # type: ignore[no-untyped-call]
```

Comment:
Could the context var be typed so we wouldn't need this warning?

Reply:
We could introduce a proxy method around this? One that returns the expected type, and raises when it is None?

Reply:
Fixed in dev, removed from this PR.